### PR TITLE
chaosctl: refine logging and error wrapping

### PR DESCRIPTION
**Pull request by STRRL**
### What problem does this PR solve?

It's the final part of https://github.com/chaos-mesh/chaos-mesh/issues/1397;

### What is changed and how does it work?

- using `errors.Wrapf()` instead of `fmt.Errorf()` (Actually I think a bunch of errors could return directly 🤪 )
- using `logr` and `klogr` for printing debug level logs into `stderr`

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by STRRL** [comment]
This PR is ready to review, PTAL @Yiyiyimu @WangXiangUSTC @YangKeao @cwen0 .

File: pkg/chaosctl/common/common.go [hunk 1]
```diff
@@ -338,14 +343,14 @@ func Log(pod v1.Pod, tail int64, c *kubernetes.Clientset) (string, error) {
 	req := c.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &podLogOpts)
 	podLogs, err := req.Stream()
 	if err != nil {
-		return "", fmt.Errorf("failed to open stream: %v", err.Error())
+		return "", errors.Wrapf(err, "faield to open log stream for pod %s/%s", pod.GetNamespace(), pod.GetName())
```
Comment (Yiyiyimu) [inline pkg/chaosctl/common/common.go:346]: s/faield/failed/g

**Review by Yiyiyimu — APPROVED** [review]
Cool! That's way more elegant!

File: pkg/chaosctl/common/common.go [hunk 2]
```diff
@@ -366,7 +371,7 @@ func checkConnForCtrlAndDaemon(ctx context.Context, daemons []v1.Pod, c *ClientS
 	}
 	ctrlMgrs, err := selector.SelectPods(ctx, c.CtrlCli, c.CtrlCli, ctrlSelector, ctrlconfig.ControllerCfg.ClusterScoped, ctrlconfig.ControllerCfg.TargetNamespace, ctrlconfig.ControllerCfg.AllowedNamespaces, ctrlconfig.ControllerCfg.IgnoredNamespaces)
 	if err != nil {
-		return fmt.Errorf("failed to SelectPods with: %s", err.Error())
+		return errors.Wrapf(err, "faield to select pod for controller-manager")
```
[the remark this answers is not in the document]
Comment (STRRL) [reply pkg/chaosctl/common/common.go]: fixed!

File: pkg/chaosctl/common/common.go (hunk 1 above)
Comment (STRRL) [reply pkg/chaosctl/common/common.go]: fixed!

**Review by Yiyiyimu — APPROVED** [review]
LGTM

**Comment by STRRL** [comment]
Hi @Yiyiyimu PTAL, I just merge the upstream/stream for conflict, this action dismisses your approval. 🤣

**Review by Yiyiyimu — APPROVED** [review]
LGTM

**Comment by Yiyiyimu** [comment]
> Hi @Yiyiyimu PTAL, I just merge the upstream/stream for conflict, this action dismisses your approval. 🤣

Ha no worries 😆

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**